<!-- Make sure launch ctrl_preapare first -->
<launch>
    <param name="/use_sim_time" value="true" />
    <node pkg="thewalkingdead" type="task7.py" name="task7" output="screen"/>
</launch>